CONTRIBUTING.md

```diff
@@ -15,35 +22,49 @@ Did you find a bug in GeoBlacklight or interested in a new feature? Make sure to
     - Explain the actual behavior
     - Provide steps to reproduce the actual behavior
 
-### GeoBlacklight Software Versioning
-GeoBlacklight follows the practice of [Semantic Versioning](https://semver.org/) for software releases. The declared semantically versioned public API includes:
- - the GeoBlacklight-Schema
- - the [public GeoBlacklight Ruby codebase classes](http://www.rubydoc.info/github/geoblacklight/geoblacklight/master/frames)
- - the GeoBlacklight JavaScript interface
- - the GeoBlacklight view interface
-
-### Contributing code or documentation
-GeoBlacklight welcomes code and documentation contributions. You do *not* need to be a "GeoBlacklight Committer" to contribute code or documentation. We follow the [pull request](https://help.github.com/articles/using-pull-requests/) model for contributing on GitHub. GeoBlacklight uses a suite of tests to express its features and protect from bugs:bug:.
+## Contributing code or documentation
```

Comment:
Maybe "Contributing to the codebase" since we have a separate documentation contribution guide